src/jsonrpc/client/pool.rs

```diff
@@ -34,8 +34,7 @@ pub fn pool_start(
     prompt: bool,
 ) -> StratisResult<()> {
     if prompt {
-        let password = prompt_password(false)?
-            .ok_or_else(|| StratisError::Msg("Password provided was empty".to_string()))?;
+        let password = prompt_password(false)?;
```

Comment:
Is it intentional allowing an empty password on pool start?

Reply:
Yes. However it is intended to address a very strange behavior that was reported to me but that I have not been able to observe myself. I may withdraw it.